web/themes/custom/novel/templates/views/eventinstance--stacked-event.html.twig

```diff
@@ -0,0 +1,8 @@
+
+<a href="{{ eventinstance_url }}" class="event-list-item-stacked arrow__hover--right-small">
+  <time class="event-list-item-stacked__content" datetime="{{ datetime_attribute }}">
+    <span class="event-list-item-stacked__date">{{ content.date }}</span>
+    <span class="event-list-item-stacked__time">{{ start_time }} - {{ end_time }}</span>
+  </time>
```

Comment:
Consider adding a more explicit title.

Currently the stacked event is a separate list item in the DOM but the content of the link within has no information about what the item is about. It is simply a date.

One approach to this would be to add [a visually hidden element](https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/3308fb2d6640c71e5b44ded179bf405360b82b61/src/styles/scss/tools/mixins.misc.scss#L11-L11) containing the event instance title.